### DISCUSSION TITLE
About Microsoft's OpenJDK

### DISCUSSION THREAD

**Discussion by vico93** [AnOpenSauceDev/FastRandom#46] [Q&A] [answered]
Is the [Microsoft build of OpenJDK](https://www.microsoft.com/openjdk) (i think this is also the brand of the JRE embedded within the launcher - see [here](https://minecraft.wiki/w/Tutorials/Update_Java)) compatible with Faster Random?

**Comment by AnOpenSauceDev** [comment]
I'll go have a check. If your JVM implements JEP 356 (which it really should), you'll be fine.

**Comment by AnOpenSauceDev** [comment, marked as answer]
The latest version of JDK 17 and 21 work just fine for me on Linux (from the portable tar.gz anyways)
